docs/source/developer_guide/get_started.rst

```diff
@@ -143,6 +146,19 @@ To document a new module, a new block must be added. For example, for the ``janu
         make clean; make html
 
 
+Notebook tutorials
+++++++++++++++++++
+
+Jupyter notebooks in ``docs/source/tutorials`` are automatically run by ``Sphinx`` using the
+`nbsphinx <https://nbsphinx.readthedocs.io/en/0.2.15/index.html>`_ extension, creating the :doc:`Tutorials </tutorials/index>`.
+
+
+These are tested before ``janus-core`` is published to PyPI, but can be tested locally by running::
+
+        cd docs
+        make clean; make tutorials
+
+
```

Comment:
I will add a paragraph with the google colab shortcuts... they are in the readme of the old repo, may need to be updated to match

Reply:
So at the moment we have the links in the Python part of the README: https://github.com/stfc/janus-core?tab=readme-ov-file#python-interface and the user guide Python section: https://stfc.github.io/janus-core/user_guide/python.html

I hadn't updated the links, but other than that, do we need them anywhere else/more prominently?

I considered moving the Python bit above the CLI in the README, since it's a little way down, but the CLI is also quite a nice way to have a go when you first install, so I'm not sure.

Reply:
let us chat tomorrow about it at the meeting, you miss the icons for the links.
separate I wonder if there is any value to offer these scripts as pure python too.